tests/auth_tests/test_apps.py

```diff
@@ -0,0 +1,58 @@
+import os
+import subprocess
+import sys
+import tempfile
+import unittest
+
+from django.db import ConnectionHandler
+
+
+class AppConfigTestCase(unittest.TestCase):
```

Comment:
AppConfigTests

"TestCase" designates that this is meant to be subclassed and doesn't contain any tests itself.